ReactAndroid/src/main/java/com/facebook/react/animated/ValueAnimatedNode.java

```diff
@@ -16,6 +16,7 @@
  * library.
  */
 /*package*/ class ValueAnimatedNode extends AnimatedNode {
+  /*package*/ String mStringValue = null;
```

Comment:
Why it can't be `protected`?

Reply:
Should work as well yeah

Reply:
@osdnk I think it's probably better to keep this restricted to package-private, or do you think AnimatedNode is likely to be extended by other packages?